### DISCUSSION TITLE
Can bot avatar/picture be displayed in telegram channels?

### DISCUSSION THREAD

**Discussion by SckyzO** [containrrr/shoutrrr#303] [Q&A] [answered]
Hello, 

I use Watchtower on multiples instances to manage my containers, it works very fine. 

1 week ago I did a new telegram integration to be notified when containers are updated. It work perfectly, but when watchtower/shoutrrr post into my channel, it don't display the bot avatar / picture. 

I don't find no information on this feature. 

It is possible to have this feature ?

**Comment by piksel** [comment, marked as answer]
No, or rather, telegram channels are more of a blog/news feed. No messages in channels have any notion of who the author was, the messages are always from the channel itself.
This means that there will be no bot name or bot image, but the channel name and image ought to be in the notifications.

If you want more of a classic chat, you can create a Group instead. You can invite `@shoutrrrbot` to the group chat to get ChatID for the group chat.

**Reply by piksel** [reply]
Actually, the author *can* be displayed, it's a channel setting, but the authors image is not added to the message.
